packages/check/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "cosmwasm-check"
-version = "2.0.0"
+version = "2.0.1"
```

Comment:
Just as a sidenote: None of the crates seem to be independently versioned, so we could version them by having the version in the workspace under `workspace.package` and just reference them.

Workspace:

```toml
[workspace.package]
version = "2.0.1"
```

Package:

```toml
[package]
version.workspace = true
```

(<https://doc.rust-lang.org/cargo/reference/workspaces.html#the-package-table>)

Reply:
Yeah, the current system was created before this cargo feature existed. Feel free to refactor whenever you get the chance